src/Enum/Status.ts

```diff
@@ -0,0 +1,15 @@
+export enum Status {
+  NOT_ENOUGH_FUNDS = "Account doesn't have enough funds to send transaction.",
+  WALLET_BUSY = 'Sending transaction is already in progress. Please wait for account to complete tx.',
+  IN_PROGRESS = 'Transaction in progress',
+  UNKNOWN_ERROR = 'An error happened',
+  OK = 'OK',
+  CLAIM_FAILED = 'Claiming: Transaction already claimed',
+  EXECUTE_FAILED = 'Execution: Unable to send the execute action',
+  TX_FAILED = 'FAILED',
+  CLAIM_PENDING = 'Claiming: Skipped - Other claiming found',
+  EXECUTE_PENDING = 'Execution: Skipped - Other execution found',
+  NOT_ENABLED = 'Claiming: Skipped - Claiming disabled',
+  ACCOUNT_BUSY = 'Claiming: Skipped - Account is busy',
+  SUCCESS = 'Success'
+}
```

Comment:
This enum contains a lot of noise, please stick to previous enum definition. Maybe decouple message from enum or use enum string as templates. I don't really see a need to keep both ACCOUNT_BUSY and WALLET_BUSY and other duplicated ones.

Reply:
Okay will keep a single status like before, no problem, but for almost similar statuses like `FAILED` for `EXECUTE` and `CLAIM` how do I manage the differences, do I use ``Claiming: ${Status.FAILED}`` for claim fail and ``Execution ${Status.FAILED}`` this way we can have single `FAILED` status in the file, I'm sorry if I don't understand it very clearly. @kosecki123